docs/quickstart.md

```diff
@@ -4,6 +4,8 @@ Note: this is a quick guide for getting something up and running. This is config
 
 ## Pre-reqs
 
+*The quickstart currently only supports macOS.*
```

Comment:
Was there a particular thing you ran into that lead to this note? I know we download binaries which are built for Mac but was curious if there was anything else

Reply:
`brew` could be (there is linux support, but not sure how prominent that is/if linux users would even want that). Other than that, no, more of trying to make it very clear what this has been tested/supported on.